### DISCUSSION TITLE
html-FundMe

### DISCUSSION THREAD

**Discussion by Proder** [Cyfrin/foundry-full-course-cu#473] [Q&A]
Is the withdraw button functioning correctly..? Asin when i tried to change the account and checked for the balance it was still showing the same balance and it even worked withdrawing from different account and then when I checked the balance it was still the same

**Comment by Niferu** [comment]
Hello @Proder 

Please link your repo or show withdraw function code associated with this issue. If you are using witdraw from course repository it should work correctly.

**Reply by Proder** [reply]
Here's my repo link: https://github.com/Proder/foundry-fundMe
And yes, despite the matter of fact that I am using the cloned html-fundMe repo, still I don't know why this is happening ? 
Here is what I am talking about:
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/97831986/6e258505-0681-40ba-8068-1d1b4a6428a2)

**Reply by Niferu** [reply]
Hi @Proder 

Your repository has to be private because I cannot open it. Change it's visibility, so I'm pretty sure I will be able to help you. But if you are really using course repo code it should work correct. Are you sure you are working with correct contract?

**Reply by Proder** [reply]
Yes! you can check it out..I am myself wondering why this is happening even when html-fundMe I am using the cloned repo and its abi and regarding FundMe contract, it is same as in the video

**Reply by Niferu** [reply]
@Proder 

Your repository contains only backend and your problem lies almost for sure in frontend, so repository containing html file etc.

**Reply by Proder** [reply]
As i said, i have cloned it from course repo itself and that is the one I'm using

**Reply by Niferu** [reply]
If you cloned repo there is no way withdraw function is corrupted. I was doing this lesson 3 times with 3 different frameworks and it worked perfectly fine, so only way it is acting up is that you have changed something or you calling wrong contract or function under your button or you are just checking balance incorrectly somehow.
